### PR TITLE
Rename labs name

### PR DESCRIPTION
**Pull request by Dawnlord**
Update images

### CODE REVIEW

**Comment by Dawnlord** [comment]
@kai33 @fionachang May you review this?

File: _data/features.yml [hunk 1]
```diff
@@ -16,14 +16,14 @@
...
-  url: "./docs.html#spotlight"
+  url: "./docs.html#spotlight-lab"
 }
 
 - {
   img: "./img/auto-animate.gif",
   tagline: "Easily transition from A to B.",
   description: "Show where objects start, and where they should go. <strong>Auto animate</strong> takes care of the rest.",
-  url: "./docs.html#auto-animate"
+  url: "./docs.html#animation-lab"
```
Comment (BCDarkKnight) [inline _data/features.yml]: Can you also rename the feature name in the description?

Comment (BCDarkKnight) [inline _data/features.yml]: Can you also rename the image files and update the path in `img`?

**Comment by Dawnlord** [comment]
The website can work on my computer but fail in checking. I will try to fix it now.

**Pull request merged**